### PR TITLE
feat: accept GET method at /quitquitquit

### PR DESCRIPTION
**Pull request by kchygoe**
Fix the issue https://github.com/GoogleCloudPlatform/alloydb-auth-proxy/issues/725

### CODE REVIEW

File: cmd/root_test.go [hunk 1]
```diff
@@ -1210,8 +1210,8 @@ func TestQuitQuitQuit(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to dial endpoint: %v", err)
 	}
-	if resp.StatusCode != http.StatusBadRequest {
-		t.Fatalf("expected a 400 status, got = %v", resp.StatusCode)
+	if resp.StatusCode != http.StatusOK {
```
Comment (nancynh) [inline cmd/root_test.go:1213]: Could you add unit test coverage for a method that's not a GET or POST as well?

**Review by nancynh — CHANGES_REQUESTED** [review]
Thanks you for the changes! Please also take a look at the failing unit tests

File: cmd/root_test.go (hunk 1 above)
Comment (kchygoe) [reply cmd/root_test.go:1213]: sure!

Comment (kchygoe) [reply cmd/root_test.go:1220]: fixed test aligned with cloud-sql-proxy

Comment (nancynh) [reply cmd/root_test.go:1220]: Thanks!

**Review by nancynh — APPROVED** [review]
LGTM, thank you!

**Pull request merged**